TypeInjections/Tool/Tool.fs

```diff
@@ -35,15 +39,24 @@ module Tool =
           [<Option('c', "clone", Required = false, HelpText = "input a git URL")>]
           url: string
           [<Option('f', "force", Required = false, HelpText = "overwrite the Old and New folders in the Output folder")>]
-          force: bool }
+          force: bool
+          [<Option('i', "ignore", Required = false, HelpText = "specify filename(s) you want to ignore")>]
+          ignore: seq<string>}
+        
+    (* TypecartProject's 2nd argument can be:
```

Comment:
What is the `2nd argument` here referring to?

Reply:
If `Required=true` then this option must be included in the command.
Ex. Since `--new` is required for the `git` command, the command line parser would throw an error at this command:
`$ typecart git --old /Users/Desktop`
Since we can run typeCart without ignoring files, I thought this option should be `required=false`